pysat/tests/classes/cls_instrument_property.py

```diff
@@ -0,0 +1,752 @@
+"""Test for instrument properties in the pysat Instrument object and methods.
+
+Note
+----
+Base class stored here, but tests inherited by test_instrument.py
+
+"""
+
+import datetime as dt
+from importlib import reload
+import logging
+import numpy as np
+import warnings
+
+import pandas as pds
+import pytest
+import xarray as xr
+
+import pysat
+from pysat.utils import testing
+from pysat.utils.time import filter_datetime_input
+
+
+class InstPropertyTests(object):
+    """Basic tests for `pysat.Instrument` properties.
+
```

Comment:
Recommend adding a list of class attributes to the docstring.  This would be really helpful since there is no local setup/teardown that effectively shows a new user what they need to include there.